.github/CONTRIBUTING.rst

```diff
@@ -29,10 +29,8 @@ everything from how to build the code to submitting a pull request. There are
 also suggestions on how you can most effectively help the project.
 
 Please be aware that our workflow does deviate slightly from the typical GitHub
-project. Issues are tracked at https://bugs.python.org/ and should be mentioned
-in pull requests using the format of ``bpo-12345`` . If your pull request is in
-response to a specific issue then please mention it in the title,
-e.g. “bpo-12345: fix comment in tokenizer.c”.
+project. Details on how to properly submit a pull request are covered in
```

Comment:
So I put that word in to make sure people read it, otherwise submitting a PR without following those guidelines won't lead to your PR being accepted outright, hence not done properly.